CHANGELOG.rst

```diff
@@ -662,7 +662,7 @@
 
 **JDBC Driver 3.2.5**
 
-- \|SNOW-33566| Added support for ``ResultSet.isLast()``, ``isBeforeFirsrt()``, and ``isAfterLast()``.
+- \|SNOW-33566| Added support for ``ResultSet.isLast()``, ``isBeforeFirst()``, and ``isAfterLast()``.
```

Comment:
Some projects don't like changing their changelogs. I'm happy to drop changes to any files, just let me know.